CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Would it make sense to make this "Delta Lake Contributor Covenant Code of Conduct"?